### DISCUSSION TITLE
is there a way to disable warnings and alarms?

### DISCUSSION THREAD

**Discussion by himse1f** [Louisvdw/dbus-serialbattery#492] [Q&A]
is there a way to disable alarms/warnings (notifications) for the whole dbus-serialbattery driver or maybe specific to each device such as /dev/ttyUSB0 or /dev/ttyUSB1, etc?

The alarms/notifications i am talking about is when you go to the remote console of the cerbogx and go to Notifications. 

my cerbogx screen is nothing but warnings and alarms and it is driving me nutz. i cant find anything in the settings or in the .py files for dbus-serialbattery

**Comment by mr-manuel** [comment]
Hello,

please check this:

- https://github.com/Louisvdw/dbus-serialbattery/discussions/482
- https://github.com/Louisvdw/dbus-serialbattery/issues/487#issuecomment-1462096548 number 2
